docs/howto/openstack/nova/server-group.md

```diff
@@ -0,0 +1,88 @@
+---
+description: >
+  Server groups provide a mechanism to group servers
+  according to a certain policy.
```

Comment:
Just to keep the recently added markdownlint testenv happy, make this a one-liner instead.

```suggestion
description: Server groups provide a mechanism to group servers according to a certain policy.
```